cloud-service-provider/azure/aks/terraform/README.md

```diff
@@ -0,0 +1,85 @@
+# OPEA applications Azure ASK deployment guide
+
+This guide shows how to deploy OPEA applications on Azure Kubernetes Service (ASK) using Terraform.
+
+## Prerequisites
+
+- Access to Azure ASK
+- [Terraform](https://developer.hashicorp.com/terraform/tutorials/azure-get-started/install-cli), [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/) and [Helm](https://helm.sh/docs/helm/helm_install/) installed on your local machine.
+- Keep the Azure subscription handy and enter the subscription id when prompted during the terraform execution.
+
+## Setup
+
+The setup uses Terraform to create ASK cluster with the following properties:
+
+- 1-node ASK cluster with 50 GB disk and `Standard_D32d_v5` SPOT instance (16 vCPU and 32 GB memory)
```

Comment:
Do we need a SPOT instance? Can this be a standard virtual machine?

Reply:
To support cost optimization, we have mentioned SPOT. Since instance is a variable, we will mention standard/SPOT in the readme file and mention about cost optimization in the variable file.